### ISSUE TITLE
Demo: Menu highlights missing in iOS

### ISSUE THREAD

**Issue by ToniBonittoGSA** [GSA/digitalgov.gov#1077]
**iOS** (Android is below)

![menu Image from iOS](https://user-images.githubusercontent.com/19961574/58713113-efe78400-838f-11e9-9e93-33781896c877.png)

***

**Android**

![menu image from Android](https://user-images.githubusercontent.com/19961574/58713130-f675fb80-838f-11e9-9081-bd3b1e73d458.jpg)
